marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/UpdateDhsModulesPermissionsCommand.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2012-2019 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.marklogic.hub.deploy.commands;
+
+import com.marklogic.appdeployer.command.AbstractCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.client.ext.modulesloader.Modules;
+import com.marklogic.client.ext.modulesloader.impl.DefaultModulesFinder;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.dataservices.UpdateDhsResourcePermissions;
+import org.apache.commons.io.FilenameUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class UpdateDhsModulesPermissionsCommand extends AbstractCommand {
+
+    @Autowired
+    private HubConfig hubConfig;
+    private UpdateDhsResourcePermissions service;
+
+    //This command has to be run after LoadHubModulesCommand is run.
+    public UpdateDhsModulesPermissionsCommand() {
+        setExecuteSortOrder(SortOrderConstants.LOAD_MODULES);
```

Comment:
This should be SortOrderConstants.LOAD_MODULES + 1 to ensure that the modules have all been loaded already.